### ISSUE TITLE
Keep horizontal position

### ISSUE THREAD

**Issue by sobrinho** [farmergreg/vim-lastplace#12]
Hi,

Would be nice to be able to keep the cursor position as well.

Example (_ is the cursor):

```
line
line _line
line
```

When we reopen the file, the cursor goes to the first position:

```
line
_line line
line
```

What you think?

One problem I see is that we would have to deal with lines where no longer large as the last position, in that case I think going to the last position would be okay.

**Comment by farmergreg** [comment]
Thanks for the suggestion! I'll take a look and see if/how this can be done.

**Comment by farmergreg** [comment]
I just took a look at this today, and the horizontal and vertical positions are indeed maintained correctly already. I tested this with Vim 7.4.1689 on a ubuntu-server virtual machine.

@sobrinho can you replicate your issue with a clean vim install? If so, please re-open this issue. Thanks!

**Comment by lixiaoquan** [comment]
I run into a similar issue and this is what I found. Tested on nvim 0.5 and vim 8.2 (2815)

1. If the last position isn't near the end of the file, the plugin goes through this path and horizontal position will be kept
https://github.com/farmergreg/vim-lastplace/blob/d522829d810f3254ca09da368a896c962d4a3d61/plugin/vim-lastplace.vim#L54
2. If the last position is near the end of the file, the plugin goes through this path and it always jump to the start of the line
https://github.com/farmergreg/vim-lastplace/blob/d522829d810f3254ca09da368a896c962d4a3d61/plugin/vim-lastplace.vim#L68

Even if I change line 68 the same way line 54 does, it still jump to the start of the line
```
execute "normal! g`\""
```
